Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -1786,6 +1790,20 @@ void NetworkConnectionToWebProcess::updateSharedPreferencesForWebProcess(SharedP
         session->protectedStorageManager()->updateSharedPreferencesForConnection(protectedConnection(), m_sharedPreferencesForWebProcess);
 }
 
+#if ENABLE(CONTENT_EXTENSIONS)
+void NetworkConnectionToWebProcess::shouldOffloadIFrameForHost(const String& host, CompletionHandler<void(bool)>&& completionHandler)
+{
+    bool wasGranted = false;
+
+    if (CheckedPtr session = networkSession()) {
+        wasGranted = session->protectedResourceMonitorThrottler()->tryAccess(host);
+        CONNECTION_RELEASE_LOG(Loading, "shouldOffloadIFrameForHost: (host=%" PUBLIC_LOG_STRING ", wasGranted=%d)", host.utf8().data(), wasGranted ? 1 : 0);
```

Comment:
The host here should be `SENSITIVE_LOG_STRING` as we don't want to see the hosts involved in customer logs.